ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-openshift-logging-6.1.yaml

```diff
@@ -0,0 +1,122 @@
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_NODE_TYPE: e2-standard-8
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FILTERS_ADDITIONAL: ""
+      FORCE_SUCCESS_EXIT: "true"
+      LOGGING_INDEX_IMAGE: quay.io/openshift-qe-optional-operators/aosqe-index:log6.1
+      OPERATORS: |
+        [
+            {"name": "loki-operator", "source": "qe-app-registry", "channel": "stable-6.1", "install_namespace": "openshift-operators-redhat","target_namespaces": "", "operator_group": "openshift-operators-redhat"},
+            {"name": "cluster-logging", "source": "qe-app-registry", "channel": "stable-6.1", "install_namespace": "openshift-logging","target_namespaces": "", "operator_group": "openshift-logging"},
+            {"name": "cluster-observability-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-cluster-observability-operator","target_namespaces": "", "operator_group": "openshift-cluster-observability-operator"},
+            {"name": "amq-streams", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators","target_namespaces": "", "operator_group": ""}
+        ]
+      TEST_ADDITIONAL: ""
+      TEST_FILTERS: ~Disruptive&;~Longduration&;~Cloudwatch&;~AWSSTS&;~GCL&;~AZWIF&;~AZMonitor&
+      TEST_IMPORTANCE: ""
+      TEST_PARALLEL: "3"
+      TEST_ROSA_LOGIN_ENV: ""
+      TEST_SCENARIOS: sig-openshift-logging
+      TEST_TIMEOUT: "15"
+    test:
+    - ref: enable-qe-catalogsource-logging
+    - ref: install-operators
+    - chain: openshift-e2e-test-logging-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity
+- as: aws-ipi-amd-f999-logging
+  cron: 8 8 29 2 *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6i.2xlarge
+      FORCE_SUCCESS_EXIT: "true"
+      LOGGING_INDEX_IMAGE: quay.io/openshift-qe-optional-operators/aosqe-index:log6.1
+      OPERATORS: |
+        [
+            {"name": "loki-operator", "source": "qe-app-registry", "channel": "stable-6.1", "install_namespace": "openshift-operators-redhat","target_namespaces": "", "operator_group": "openshift-operators-redhat"},
+            {"name": "cluster-logging", "source": "qe-app-registry", "channel": "stable-6.1", "install_namespace": "openshift-logging","target_namespaces": "", "operator_group": "openshift-logging"},
+            {"name": "cluster-observability-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-cluster-observability-operator","target_namespaces": "", "operator_group": "openshift-cluster-observability-operator"},
+            {"name": "amq-streams", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators","target_namespaces": "", "operator_group": ""}
+        ]
+      TEST_FILTERS: Disruptive;Longduration;Cloudwatch;74397;pico;PodDisruptionBudget;~tlsSecurityProfile&
```

Comment:
I would suggest to create a separate job to only run `Disruptive` cases as these cases may make the cluster broken, and once the cluster is broken, other test cases will be affected and the test result may not be accurate.

Reply:
@QiaolingTang All cases test pass.  As we excluded the  ~tlsSecurityProfile&. I didn't hit error.

Reply:
https://docs.google.com/spreadsheets/d/1z68eUr7lyqIK_9A6TuZ3vvGa8qCbI3k_pvAnTrwzIes/edit?gid=181844650#gid=181844650
https://docs.google.com/spreadsheets/d/1z68eUr7lyqIK_9A6TuZ3vvGa8qCbI3k_pvAnTrwzIes/edit?gid=887175090#gid=887175090

Reply:
@anpingli Although you have some successful triers. But disruptive cases can make the cluster broken, that's why we add the tag to these cases. I still suggest to create a seperate job to run disruptive cases.

Reply:
Ok. I remove the keyword Disruptive.   I will create a dedicate jobs when the tlsProfile cases are fixed.